src/shared/filters/all-exceptions.filter.ts

```diff
@@ -24,11 +24,10 @@ export class AllExceptionsFilter<T> implements ExceptionFilter {
   }
 
   catch(exception: T, host: ArgumentsHost): any {
-    const ctx = host.switchToHttp();
-    const req: Request = ctx.getRequest<Request>();
-    const res: Response = ctx.getResponse<Response>();
+    const ctx = GqlArgumentsHost.create(host).getContext();
+    const req = <Request>ctx.req;
+    const res = <Response>ctx.req.res;
 
-    const path = req.url;
```

Comment:
Consider if we should return `query` in the response.

Reply:
@FelipeCO14 
Doing this: `const query = req.body.query;` we get the following response:
```
"query": "query myMeQuery{\n    me{\n      id,\n      name\n  }\n}",
```

From the following query:
```
query myMeQuery{
    me{
      id,
      name
  }
}
```

Is that enough, or do you want me to just get the query name from the whole query string?